src/core/ext/transport/inproc/OWNERS

```diff
@@ -0,0 +1,3 @@
+set noparent
```

Comment:
Let's find a third owner before setting noparent (since authors can't self-approve, this leaves a bus factor of one for any changes you or I author)